bin/rpi-dump-edid

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+# rpi-dump-edid
+#
+# Copyright (C) 2017 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# This is a simple tool to get the raw EDID of a screen attachd to the RPi.
+# The file is output to the /boot/ partition for easy access.
+#
+# This tool was used for all screen data under kano-settings/res/edid/raw
+# Requires sudo, RPi, and a screen.
+
+
+import os
+import sys
+
+from kano.utils.shell import run_cmd
+
+from kano_settings.paths import RAW_EDID_NAME_PATTERN
+
+
+RC_SUCCESS = 0
+RC_WRONG_PERMISSIONS = 1
+RC_ERR_SUBPROCESS = 2
+
+
+def main():
+    """
+    Main program.
+    """
+
+    # Get the screen model name.
+    out, err, rc = run_cmd('tvservice --name')
+    if rc:
+        sys.stderr.write(
+            'Error: Could not get the screen model!\n rc = {}\n err = {}\n'.format(rc, err)
+        )
+        return RC_ERR_SUBPROCESS
+
+    # If the EDID is corrupt, the proc might return '[E] No device present'.
+    if 'device_name' in out:
+        model = out.strip('device_name=').strip()
```

Comment:
Given that we almost certainly have a kano-settings function to do read the name, do we want to use that instead?

Reply:
Great point. That script started as a standalone tool, but forgot to integrate it with the system properly after I moved it to `bin/`. I'll look to move most of that functionality in kano-settings instead. Thanks for pointing that out 👍